src/opensuse/15.6/helix/amd64/Dockerfile

```diff
@@ -27,8 +27,8 @@ RUN zypper ref \
         lldb-devel \
         llvm \
         lttng-ust-devel \
-        python3-devel \
-        python3-pip \
+        python311-devel \
+        python311-pip \
```

Comment:
Why not install Python 3.12 anyway?

Reply:
Because 311 is the newest version available in OpenSuse Leap 15.6

Reply:
But I see this:

```
3bc4cf5341dc:/ # zypper install python312-devel
Loading repository data...
Reading installed packages...
Resolving package dependencies...

The following 5 NEW packages are going to be installed:
  libexpat1 libpython3_12-1_0 pkg-config python312-base python312-devel

5 new packages to install.
```

Reply:
Hmmm, that is what I get for trying to lookup the available packages on https://software.opensuse.org/package versus directly in the image.  I updated to 312 in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1244/commits/c6f8d87db4d381aadb3af5e35cee10ad4fdd98c9